### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by Gleeeds**
Gary

### CODE REVIEW

File: vscode_client_2/README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
 
-**NOTE** The person you send the message to must be logged in to the same mainframe server you put in extension settings
-
-13. Press enter to send your message
+1. Open Visual Studio Code
+2. Install the Mainframe messenger extension
+3. In extension settings under "zowe profile", enter the name of your Zowe profile 
+4. Enter your mainframe credentials
+**Note:** To use a custom name on your messages, enter the name under "custom name"
+5. Under "host", enter the URL of your mainframe connection in format http(s)://host(port)
```
Comment (zeibura) [inline vscode_client_2/README.md:17]: These are configuration steps - they should be in Getting Started.

File: vscode_client_2/README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
 # Mainframe Messenger for VS Code
 
-Mainframe messenger is an easy way how you can send messages from Visual Studio Code to another user on TSO on green screen. The user you send messages to must be connected to a specific mainframe connection what you configure in extension settings. Mainframe messenger has Zowe conformance and uses Zowe profile to connect to mainframe.
+Mainframe messenger is an easy way to send messages from Visual Studio Code to TSO users. Users must be connected to a specific mainframe connection that you configure in extension settings. Mainframe messenger has Zowe conformance and uses Zowe profile to connect to mainframe.
+
+##Prerequisites
+
+1. Java version 11
+2. Zowe zosmf profile with credentials defined
```
Comment (Ike-Anwalk) [inline vscode_client_2/README.md]: test comment: change to z/OSMF

File: vscode_client_2/README.md [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
-4. In extension settings, write the name of your Zowe profile under "zowe profile"
-5. Write your mainframe username and password in "username" and "password"
-6. If you want use a custom name on your messages, write the name under "custom name"
-7. Under "host" write the URL of your mainframe connection in format http(s)://host(port)
-8. Create a new file in your VS Code workspace with the extension .txt
-9. Write your message in the file 
-
-***NOTE!!!!*** Your message cannot be longer than 80 characters
+1. Java version 11
+2. Zowe z/osmf profile with credentials defined
```
Comment (Ike-Anwalk) [inline vscode_client_2/README.md:8]: test comment: change to z/OSMF

File: vscode_client_2/README.md [hunk 4]
```diff
@@ -1,29 +1,29 @@
 # Mainframe Messenger for VS Code
 
-Mainframe messenger is an easy way how you can send messages from Visual Studio Code to another user on TSO on green screen. The user you send messages to must be connected to a specific mainframe connection what you configure in extension settings. Mainframe messenger has Zowe conformance and uses Zowe profile to connect to mainframe.
```
Comment (Ike-Anwalk) [inline vscode_client_2/README.md:3]: proposed change: "how you can" to "to"

Comment (Ike-Anwalk) [inline vscode_client_2/README.md:3]: cut 'on green screen'

Comment (Ike-Anwalk) [inline vscode_client_2/README.md:3]: uses 'the' Zowe profile...

**Pull request closed without merging**